doc/api/cli.md

```diff
@@ -51,6 +51,53 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run <command> [<arguments>]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+This runs a specified command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+`node run` prepends `./node_modules/.bin`, relative to the current
+working directory, to the `PATH` in order to execute the binaries from
+dependencies.
+
+For example, the following command will run the `test` script of
+the `package.json` in the current folder:
+
+```console
+$ node run test
+```
+
+You can also pass arguments to the command. Any argument after the script
+name will be appended to the script:
+
+```console
+$ node run test --verbose
+```
+
+#### Intentional limitations
+
+`node run` is not meant to match the behaviors of `npm run` or of the `run`
+commands of other package managers. The Node.js implementation is intentionally
+more limited, in order to focus on top performance for the most common use
+cases.
+Some features of other `run` implementations that are intentionally excluded
+are:
+
+* Searching for `package.json` files outside the current folder.
+* Prepending the `.bin` or `node_modules/.bin` paths of folders outside the
+  current folder.
```

Comment:
Without the first two bullet points here, it could mean that it won't work with monorepos.

Since this feature is experimental, I suppose it could be added later without semver major so not a blocker now.

Reply:
IMO it should not "work with monorepos" in that it should not traverse the parent directories in search of a "workspace root". If you take a look at the way different tooling is doing this today it is different enough that I don't think Node.js should be attempting to bridge those gaps. AFAIK no node features today know anything about "js monorepos" and I would prefer to keep it that way. IMO this should *only* look for the `package.json` in the cwd and error if not found (I read the code a week or to ago now so do not remember if that is what it does, but I thought that was it).

Reply:
I added this to the code as `TODO`. This is not the end product.